docs/pages/management/dynamic-configuration.mdx

```diff
@@ -0,0 +1,272 @@
+---
+title: Using Dynamic Configuration Resources
+description: An introduction to Teleport's dynamic configuration resources, which make it possible to apply settings to remote clusters using infrastructure as code.
+tocDepth: 3
+---
+
+This section includes guides for managing Teleport's dynamic configuration
+resources, which make it possible to adjust the behavior of your Teleport
+cluster as your infrastructure changes.
+
+## What is a dynamic configuration resource?
+
+There are two ways to configure a Teleport cluster:
+
+- **Static configuration file:** At startup, a Teleport process reads a
+  configuration file from the local filesystem (the default path is
+  `/etc/teleport.yaml`). Static configuration settings control aspects of a
+  cluster that are not expected to change frequently, like the ports that
+  services listen on. 
+- **Dynamic configuration resources:** Dynamic configuration resources control
+  aspects of your cluster that are likely to change over time, such as roles,
+  local users, and registered infrastructure resources.
+
+This approach makes it possible to incrementally adjust your Teleport
+configuration without redeploying Teleport instances. 
+
+```mermaid
+flowchart LR
+subgraph backend["Cluster state backend"]
+  direction LR
+  resources["Dynamic config resources"]
+end 
+
+subgraph cluster[Teleport cluster]
+  subgraph auth[Auth Service]
+    conf1["Static config"]
+  end 
+  subgraph proxy[Proxy Service]
+    conf2["Static config"]
+  end 
+end
+
+auth--"Reads, writes"-->backend
+subgraph clients["API clients"]
+  terraform["Terraform Provider"]
+  operator["Kubernetes Operator"]
+  tctl
+end
+terraform & operator & tctl-- "API calls" -->proxy
+proxy-.->auth
+```
+
+Teleport borrows the concept of a dynamic configuration resource from the REST
```

Comment:
I don't know that this sentence is necessarily true, and putting that aside I don't think it adds much. Can we just remove it?

Reply:
I moved it along with some other introductory text from the configuration resource reference, but this feedback makes sense—I'll remove it.